requirements/AOCS-GNC_AURORA_0001/taste_project/DataView.aadl

```diff
@@ -0,0 +1,636 @@
+--------------------------------------------------------
```

Comment:
This file is automatically generated - please remove

Reply:
Done